content/administration/upgrade.rst

```diff
@@ -1,264 +1,367 @@
 
-.. seealso::
-   - :ref:`upgrade/sla`
-
-.. _upgrade/process-workflow:
-
-Process workflow
-================
-
-The upgrade process in a nutshell:
-
-#. You create a test upgrade request.
-#. Odoo processes the request automatically by running the database through an upgrade script, which
-   takes between 20 and 120 minutes.
-#. Odoo delivers a test database.
-#. You test your database for possible discrepancies (see :ref:`upgrade/test-guidance`).
-#. If there are any discrepancies, you report them to the Upgrade support team via the help portal
-   (see :ref:`upgrade/test-assistance`).
-#. We fix the issues and send you a new test database.
-#. Once you have completed the testing and are happy with the result, you decide on a date and time
-   when you stop users from accessing Odoo, freeze all data entries, and create an upgrade request
-   for the production upgrade.
-#. Odoo delivers the production database through the automated process.
-#. You restore it in your Production environment a few short hours later and continue working on the
-   newly upgraded database (this is done automatically on Odoo Online).
 
-.. seealso::
-   - :doc:`Upgrade process for Odoo Online <upgrade/odoo_online>`
-   - :doc:`Upgrade process for Odoo.sh <upgrade/odoo_sh>`
-   - :doc:`Upgrade process for On-Premise <upgrade/on_premise>`
+Upgrading in a nutshell
+-----------------------
 
-.. _upgrade/testing-phase:
+#. Request an upgraded test database (see :ref:`obtaining an upgraded test database
+   <upgrade/request-test-database>`).
 
-Testing
-=======
+#. Thoroughly test the upgraded database (see :ref:`testing the new version of the database
+   <upgrade/test_your_db>`).
 
-This phase allows you to review an upgraded version of your database without affecting your
-production database in any way. We suggest that you run the test upgrade process at least once, but
-you can do it as many times as you need (one at a time).
+#. Report any issue encountered during the testing to Odoo via the `support page
+   <https://odoo.com/help?stage=migration>`__.
 
-Once you receive your upgraded test database, check that all data, processes, and functionality are
-still correct and working as expected.
+#. (If applicable) : upgrade the source code of your custom module to be compatible with the new
+   version of Odoo.
 
-If you do find discrepancies, :ref:`report your issues <upgrade/test-assistance>` and :ref:`request
-a new test database <upgrade/test-db-request>` when the reported issues are fixed in the upgrade
-script.
+#. Once all issues are resolved and you are confident that the upgraded database can be used as
+   your main database without any issue, plan the upgrade of your production database.
 
-If you do not find any discrepancies, you can move on to the upgrade of your production database.
+#. You or Odoo upgrade the production database, rendering it unavailable for a few hours
+   (see :ref:`upgrading the production database <upgrade/upgrade-prod>`).
```

Comment:
I'd say it similar to how it is in the test part
Also, let's not say it takes hours :see_no_evil: 
I know it's already there in the current doc, but it can scare people and it's usually not true, most dbs are small and take at most 15-20 mins (tho I wouldn't say it either as each db is different).
```suggestion
# Request the upgraded for the production database, rendering it unavailable for the time it takes to complete the process (see :ref:`upgrading the production database <upgrade/upgrade-prod>`).
```
For the rest, this new section looks good to me :+1:
Thanks for adding it back and adapt it to the new doc schema!!